content/installation/system_requirements.md

```diff
@@ -88,3 +80,9 @@ source control) needs to be installed (for instance, Maven or Rake).
 * **Subversion** >= 1.6.11
 * **TFS** SDK 14.0.3 (TFS 2012, 2013, 2015 and Visual Studio Team Services are supported by GoCD)
 * **Perforce** >= 2016.1
+
+### Java Dependencies for GoCD (Server and Agent)
+
+Starting September 2019, GoCD will support the **last 3 versions** of Java release. All the docker images and installers for Windows and OSX are bundled with the supported Java version.
```

Comment:
I think we should be specific with the version we support for each release of GoCD. When we say `last 3 versions` it would be difficult to interpret the version of Java required for an older version of GoCD.

Reply:
Updated to specify the exact Java version supported.

Reply:
I agree. We should also do this in the release notes.

Now it says: "GoCD requires Java Runtime Environment (JRE) version 9 (JRE 11 and above recommended)."

Maybe we can say: "GoCD works with Java Runtime Environment (JRE) version 9, 10 and 11 (recommended)." (or whatever the right versions are.

Reply:
> GoCD works with Java Runtime Environment (JRE) version 9, 10 and 11 (recommended).
Sounds better!

I was thinking,
```
This version of GoCD works with Java Runtime Environment (JRE) versions 8, 10, 11 and 12(recommended).

Given Java has moved to a 6-month release cycle, GoCD will work with the lastest 3 available versions of Java at the time of the release. Please refer this [blogpost](https://www.gocd.org/2019/05/21/official-stance-on-java/) for our official stance on supported Java versions.
```

Reply:
@maheshp - I was thinking about recommending the latest LTS(which is jdk 11) instead of the latest release(12) - WDYT?